UnityProject/Assets/Scripts/Systems/Occupations/OccupationCustomEffects/AnotherThing.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnotherThing : OccupationCustomEffectBase
```

Comment:
this is a developer class for testing live, I imagine. Doesnt look like it is needed anymore.